webapp/src/app.tsx

```diff
@@ -3906,6 +3907,7 @@ export class ProjectView
             this.editor == this.textEditor && this.state.errorListState,
             'full-abs',
         ];
+        this.rootClasses = rootClassList;
```

Comment:
I'm a little hesitant on this because there are just so many classes this adds / wide variety of scenarios that make it a little hard to know we've tested everything. Could we pull out just the ones that we definitely need here into a list for now, and maybe see about expanding / cleaning it up later? I think that's invertedTheme, hc

Reply:
yeah that sounds good! I think hc actually is handled in a different way, so I won't touch it here